code/modules/security_levels/security_levels.dm

```diff
@@ -53,7 +53,8 @@ GLOBAL_VAR_INIT(security_level, SEC_LEVEL_GREEN)
 
 			if(SEC_LEVEL_RED)
 				if(GLOB.security_level < SEC_LEVEL_RED)
-					minor_announce(CONFIG_GET(string/alert_red_upto), "Attention! Code red!", TRUE, custom_alert_sound = 'sound/misc/notice4.ogg')
+					minor_announce(CONFIG_GET(string/alert_red_upto), "Attention! Code red!", TRUE)
+					sound_to_playing_players('sound/misc/notice4.ogg')
```

Comment:
why isnt this custom sound in a datum/security_level entry so we can set it manually per level?

Reply:
because it only ever plays when red alert is set

Reply:
wait im stupid i think I see what you mean